content/concepts/introduction/intro-to-consensus.md

```diff
@@ -0,0 +1,63 @@
+# Introduction to Consensus
+
+_Consensus_ is the most important property of any decentralized payment system. In traditional centralized payment systems, one authoritative administrator gets the final say in how and when payments occur. Decentralized systems, by definition, don't have an operator to do that. Instead, decentralized systems like the XRP Ledger define a set of rules all participants follow, so every participant can agree on the exact same series of events and their outcome at any point in time. We call this set of rules a _consensus protocol_.
+
+
+## Consensus Protocol Properties
+
+The XRP Ledger uses a consensus protocol unlike any digital asset that came before it. This protocol, known as the XRP Ledger Consensus Protocol, is designed to have the following important properties:
+
+- Everyone who uses the XRP Ledger can agree on the latest state, and which transactions have occurred in which order.
+- All valid transactions are processed without needing a central operator or having a single point of failure.
+- The ledger can make progress even if some participants join, leave, or behave inappropriately.
+- If too many participants are unreachable or misbehaving, the network fails to make progress rather than diverging or confirming invalid transactions.
```

Comment:
This point reminds me of something David and Nik mentioned in a lunch and learn. They said that these are the operating principles of the XRP Ledger, in this order of priority: Correctness, Agreement, Forward Progress. This bullet point is an example of these principles and priorities in action.

Because these principles are also very grok-able, I thought I'd mention them - perhaps there's a place for them in an intro article or elsewhere. Though I know I'm supposed to be looking for ways to trim this down. 😹